src/pages/Chat.vue

```diff
@@ -202,6 +205,16 @@ export default {
     }
   },
   computed: {
+    nameColor () {
+      // Get color
+      if (this.address) {
+        const addr = new Address(this.address)
+        const { hue, saturation } = addressColor(addr)
+        return `color: hsl(${hue}, ${saturation * 100}%, 60%);`
```

Comment:
BTW, You can use maps or something here I believe.  Bit better than string substitution